sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -476,7 +481,7 @@ def get_tx_power(self):
         return tx_power
 
     def get_tx_power_support(self):
-        return not self.is_flat_memory()
+        return not self.is_flat_memory() and self.xcvr_eeprom.read(consts.TX_POWER_SUPPORT_FIELD)
```

Comment:
i don't think you can read this field which is inside a reg-group field directly